models/UNet2dExample.model.yaml

```diff
@@ -70,53 +50,47 @@ outputs:
-    dependencies: conda:./environment.yaml
-
-training:
-  setup:
-#    meta_sampler:
-#      spec:
-#      kwargs:
-    samplers:
-      - spec: https://github.com/bioimage-io/python-bioimage-io/blob/b72f9743f25367ee5d70d699551bd03013765b1f/specs/samplers/SequentialSamplerAlongDimension.sampler.yaml
-        kwargs: {sample_dimensions: [0, 0]}
-        readers:
-          - spec: https://github.com/bioimage-io/python-bioimage-io/blob/b72f9743f25367ee5d70d699551bd03013765b1f/specs/readers/BroadNucleusDataBinarized.reader.yaml
-            transformations:
-              - spec: https://github.com/bioimage-io/pytorch-bioimage-io/blob/8959dab1236a72593613a5c89ea973c2f5612aea/specs/transformations/NormalizeZeroMeanUnitVariance.transformation.yaml
-                kwargs: {apply_to: [0]}
-    preprocess: []
-    postprocess:
-      - spec: https://github.com/bioimage-io/pytorch-bioimage-io/blob/8959dab1236a72593613a5c89ea973c2f5612aea/specs/transformations/Sigmoid.transformation.yaml
-        kwargs: {apply_to: [0]}
-      - spec: https://github.com/bioimage-io/pytorch-bioimage-io/blob/8959dab1236a72593613a5c89ea973c2f5612aea/specs/transformations/Cast.transformation.yaml
-        kwargs: {apply_to: [1], dtype: float32}
-    losses:
-      - spec: https://github.com/bioimage-io/pytorch-bioimage-io/blob/8959dab1236a72593613a5c89ea973c2f5612aea/specs/transformations/BCELoss.transformation.yaml
-    optimizer:
-      source: torch.optim.Adam
-      required_kwargs: [params]
-      optional_kwargs: {lr: 0.002}
-    # validation:
-    #   - {}
-
-  source: pybio.torch.training.simple.simple_training
-  required_kwargs: [pybio_model]
-  optional_kwargs: {n_iterations: 25, batch_size: 4, num_workers: 2, out_file: ./weights.pytorch}
-  # enable different ways of specifying the dependencies.
-  # this would hold all training dependencies, e.g. as a frozen conda environment
-  # or as a pom.xml
-  dependencies: # this is a file to the dependencies
-      conda:../environment.yaml
-  description: Train the unet via binary cross entropy
+# language + framework specify runner
+language: python
+framework: pytorch
+weights_format: pytorch  # specifies how the weights are stored
+
+source: ./unet2d.py:UNet2d  # optional depending on weights_format
+sha256: d6e2e4cf0562237ed0f8fbc8a518fe7047e8e75a15a8a64437bc38d7458f3687   # optional
+kwargs: {input_channels: 1, output_channels: 1}
+dependencies: conda:./environment.yaml
+
+weights:
+    - id: default
+      name: Default Weights
+      description: "Weights from DataScienceBowl"
+      authors: ["author1", "author2"]
+      covers: []
+      source: https://zenodo.org/record/3446812/files/unet2d_weights.torch
+      sha256: e4d3885bccbe41cbf6c1d825f3cd2b707c7021ead5593156007e407a16b27cf2
+      timestamp: 2019-12-11T12:22:32Z  # ISO 8601
+      test_inputs: my_default_test_input.npy  # language + extension defines memory representation
+      test_outputs: my_default_test_output.npy
```

Comment:
This should be a list

Reply:
Yes, agreed. We forgot to formalize this when turning the conversation to normalization. I made a separate PR: https://github.com/bioimage-io/configuration/pull/34